app/views/layouts/_flash.html.erb

```diff
@@ -1,3 +1,6 @@
-<% flash.each do |type, message| %>
-  <%= content_tag(:div, message, class: "flash #{type} text-green-500") %>
-<% end %>
\ No newline at end of file
+<div class="flashes">
```

Comment:
Is this LRA-431 and LRA-444?

Reply:
I added a comment to LRA-431 and LRA-444